components/micro-gateway-core/pom.xml

```
@@ -38,6 +38,23 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.org.everit.json</groupId>
+            <artifactId>org.everit.json.schema</artifactId>
+            <scope>compile</scope>
```

IMO scope should be `provided`
https://howtodoinjava.com/maven/maven-dependency-scopes/
```suggestion
            <scope>provided</scope>
```